go/vt/vitessdriver/driver_test.go

```diff
@@ -38,9 +38,7 @@ import (
 	"vitess.io/vitess/go/vt/vtgate/grpcvtgateservice"
 )
 
-var (
-	testAddress string
-)
+var testAddress string
```

Comment:
This file got run through my more strict autoformatter but I can revert if needed.